sources/packages/backend/apps/api/src/app.module.ts

```diff
@@ -15,7 +15,7 @@ import { AppInstitutionsModule } from "./app.institutions.module";
 import { ClientTypeBaseRoute } from "./types";
 import { AppStudentsModule } from "./app.students.module";
 import { AppSupportingUsersModule } from "./app.supporting-users.module";
-import { ZeebeModule } from "@sims/services";
+import { GlobalHttpModule, ZeebeModule } from "@sims/services";
```

Comment:
Just because it is global, the name starts with Global?, I would suggest AppHttpModule.

Reply:
IMO the problem is that HttpModule is already taken and we are just creating a wrapper around it. Since it can be also used in some library modules, I am not sure if it makes sense to have it prefixed with "App".